documentation/style-guide.rst

```diff
@@ -79,6 +84,11 @@ reST
    used to produce Python documentation.  When spelled out, it is
    always one word and both forms start with a lowercase 'r'.
 
+time zone
+   Spell out as two words when talking about the real-world concept.
+   Only spell as one word when referring to a Python term like a module,
+   class, or argument.
```

Comment:
Shouldn't they have markup like `:mod:` or similar in this case?
If that's the case, "timezone" should never appear in plain text.
Not sure if it's worth clarifying this.

Reply:
> Not sure if it's worth clarifying this.

Anyone reading the devguide style guide will probably already know how to mark up classes, so I'm not sure it's worth complicating the current wording.

Reply:
What I meant is:
* talking about the concept? Always use two words and no markup.
* talking about the module/class/arg? Always use one word and the appropriate markup.

This implies that "timezone" will never appear in plain text and "time zone" will never appear within markup.

IOW, the fact that you are using markup or not already determines whether to use one or two words, without having to think if you are referring to a module/class/argument or not.

Reply:
Sure, if it does not clutter the wording too much, I'm fine with such a change.